app/routes/how-can-i-help.community.tsx

```diff
@@ -1,14 +1,141 @@
 import {MetaFunction} from '@remix-run/node'
+import DropDown from '~/components/DropDown'
+import HelpItem from '~/components/HowCanIHelp/HelpItem'
+import CategoryCarousel from '~/components/HowCanIHelp/CatgoryCarousel'
 import Base from '~/components/HowCanIHelp/Base'
+import {useEffect} from 'react'
+import VideoImg from '~/assets/video_thumbnail.png'
+import PlayIcon from '~/components/icons-generated/Play'
 
 export const meta: MetaFunction = () => {
   return [{title: 'How Can I Help? - AISafety.info'}]
 }
 
+const InPerson = () => (
+  <>
+    <div className="padding-bottom-24">
+      <div className="flexbox padding-bottom-56">
+        <h2 className="teal-500 padding-bottom-40" id="new-to-aisafety">
+          In person communities
+        </h2>
+        <p className="grey default">
+          In person communities provide general purpose support, networking, and socializing for
+          anyone interested in contributing to AI safety.
+        </p>
+        {/* TODO make this only appear in left part of page */}
+      </div>
+      <HelpItem
+        title="AI safety-specific coommunities"
+        links={[
+          {
+            title: 'Our list of in-person communities',
+            action: 'https://www.aisafety.com/communities',
+          },
+        ]}
+      >
+        <p className="grey default padding-bottom-24">
+          There are about 60 in-person AI Safety communities across the globe, mostly in major
```

Comment:
safety*

Reply:
I was not sure about this one, but Claude is on your side so I will change it. See how quickly our mistrust of AI erodes! Claude knows best!

Reply:
🙃